Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-
+gemspec
 gem 'pry'
```

Comment:
I'm not sure what to do about these gems; should I leave them here or move them to the gemspec file? As dev or prod dependencies?